src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -51,6 +51,16 @@
     "NAME": "Arcane Weapon",
     "DESCRIPTION": "At the end of each turn, deal !D! stsjorbsmod:Attack\u00a0Damage to a random enemy."
   },
+  "stsjorbsmod:Assertion": {
+    "NAME": "Assertion",
+    "DESCRIPTION": "Deal ",
```

Comment:
It's preferable to use magicNumbers rather than string concatenation for filling in dynamic values; it's more localization-friendly. Combined with the feedback above about using `getRawDynamicDescriptionSuffix()`, I'd recommend something like:

```jsonc
    "DESCRIPTION": "Deal damage equal to !M! times the number of skills played this act, up to !stsjorbsmod:metaMagic! damage.",
    "EXTENDED_DESCRIPTION": [
      " NL (Deals !D! damage, !stsjorbsmod:urMagic! skills played.)"
    ]
```

Reply:
To do the skill/skills pluralization, you could still use 2 extended description options, but it's more localization friendly to accept a little repetition to avoid the concatenation, so something like this:

```jsonc
    "EXTENDED_DESCRIPTION": [
      " NL (Deals !D! damage, !M! skills played.)",
      " NL (Deals !D! damage, !M! skills played.)"
    ]
```

...and have `getRawDynamicDescriptionSuffix return one or the other based on `magicNumber` being == 1